package.json

```diff
@@ -18,43 +18,46 @@
     "commit": "git-cz",
     "dev": "NODE_ENV=development styleguidist server",
     "build": "NODE_ENV=production styleguidist build",
-    "compile": "yarn compile:clean; yarn compile:types && yarn compile:code",
+    "compile": "pnpm compile:clean; pnpm compile:types && pnpm compile:code",
     "compile:code": "rollup --config ./etc/rollup/config.js",
     "compile:clean": "rm -rf es cjs types",
     "compile:types": "tsc --p ./tsconfig.types.json",
     "test": "react-scripts test --env=jest-environment-jsdom-sixteen",
     "format": "prettier -l --write \"**/*.{ts,tsx,js,jsx,json,less,css,md}\"",
-    "format:fix": "yarn format --write",
+    "format:fix": "pnpm format --write",
     "lint": "eslint 'src/**/*.{ts,tsx}'",
-    "lint:fix": "yarn lint --fix",
-    "typecheck": "tsc --noEmit",
+    "lint:fix": "pnpm lint --fix",
+    "typecheck": "tsc --noEmit --types node,jest",
```

Comment:
this is a workaround for `yarnclean` but not ideal...

Reply:
I'm wondering whether there is an equivalent feature forthcoming. I think yarnclean was unique to yarn though...